package.json

```diff
@@ -91,25 +91,25 @@
     "@testing-library/react": "12.1.2",
     "asar": "2.1.0",
     "autodll-webpack-plugin": "0.4.2",
-    "axios": "0.21.1",
+    "axios": "0.24.0",
     "babel-eslint": "10.1.0",
     "babel-loader": "8.0.4",
     "babel-plugin-react-intl": "3.0.1",
     "bufferutil": "4.0.1",
     "cache-loader": "4.1.0",
-    "chai": "4.2.0",
+    "chai": "4.3.4",
     "chalk": "4.1.0",
     "concurrently": "5.3.0",
     "cross-env": "7.0.2",
     "css-loader": "2.0.1",
     "cucumber": "6.0.5",
     "cucumber-pretty": "6.0.0",
     "del": "6.0.0",
-    "electron-chromedriver": "13.0.0",
+    "electron-chromedriver": "16.0.0",
     "electron-connect": "0.6.3",
     "electron-devtools-installer": "3.2.0",
-    "electron-packager": "15.2.0",
-    "electron-rebuild": "1.11.0",
+    "electron-packager": "15.4.0",
+    "electron-rebuild": "2.0.1",
     "eslint": "7.10.0",
```

Comment:
in confluence we have here 2.3.5 which one is correct?

Reply:
Unfortunately `2.3.5` does not work. Let me update the confluence document. Thank you!